samples/chat-demo/main.go

```diff
@@ -281,7 +281,7 @@ func run() error {
 	if args.Verbose {
 		fmt.Println("Stopping server.")
 	}
-	ctx.Done()
+	node.Stop()
 	wg.Wait()
```

Comment:
Apparently, we don't need this `wg` anymore.

Reply:
Good point. Fixed.